test/unit/plugins/guests/linux/cap/mount_smb_shared_folder.rb

```diff
@@ -44,6 +44,8 @@
       allow(comm).to receive(:execute).with(any_args)
       allow(machine).to receive(:guest).and_return(guest)
       allow(guest).to receive(:capability).with(:shell_expand_guest_path, mount_guest_path).and_return(mount_guest_path)
+      allow(ENV).to receive(:[]).and_return(false)
```

Comment:
Should include a `with` on this to restrict matching (`allow(ENV).to receive(:[]).with("VAGRANT_DISABLE_SMBMFSYMLINKS").and_return(false)`)